src/components/List/AssetsList.vue

```diff
@@ -116,20 +116,20 @@ export default {
                 this.$proxProvider.getMosaic(el.mosaicId.id).subscribe(
                   response => {
                     response.id = el.mosaicId.id.toHex()
-                    this.mosaicArr.push(response)
+                    this.assetArr.push(response)
                   },
                   err => {
-                    console.error('Mosaic not found')
+                    console.error('Asset not found')
```

Comment:
`console` is a good tool but some browsers don't support it. It's not a big deal though since it won't make the app crash but if we are to use a logger, we should consider using `https://github.com/ptkdev/ptkdev-logger`.

It's a library that wraps the console.log and have more informative (thru colors and label) logging mechanism.

Reply:
Sounds good, will work on that in a different PR